ashley/src/com/badlogic/ashley/core/Family.java

```diff
@@ -142,10 +147,24 @@ public final Builder one (Class<? extends Component>... componentTypes) {
 		 */
 		@SafeVarargs
 		public final Builder exclude (Class<? extends Component>... componentTypes) {
-			exclude = ComponentType.getBitsFor(componentTypes);
+			exclude.or(ComponentType.getBitsFor(componentTypes));
 			return this;
 		}
 
+		/**
+		 * @param families are concat with Family in builder (one, all and exclude)
+		 * @return A Builder singleton instance to get a family
+		 */
+        @SafeVarargs
+        public final Builder concat (Family... families) {
+            for(Family family : families){
+                one.or(family.one);
+				all.or(family.all);
```

Comment:
Can this formatting be altered to be consistent?

Reply:
done :D